analysis_template/plhld/production/weights.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+
+"""
+Column production methods related to generic event weights.
+"""
+
+from columnflow.production import Producer, producer
+from columnflow.production.pileup import pu_weight
+from columnflow.production.normalization import normalization_weights
+from columnflow.util import maybe_import
+
+ak = maybe_import("awkward")
+
+
+@producer(
+    uses={normalization_weights, pu_weight},
+    produces={normalization_weights, pu_weight},
+    shifts={"minbias_xs_up", "minbias_xs_down", "scale_up", "scale_down", "pdf_up", "pdf_down"},
```

Comment:
the implementation of shifts has changed, probably needs changes in the config

Reply:
Done :+1: